destination/config.go

```diff
@@ -0,0 +1,24 @@
+package destination
+
+//go:generate paramgen -output=paramgen.go Config
+
+type Config struct {
```

Comment:
This is looking better now! Although we have the same fields also in the source config, how about we extract them into a separate struct and reuse them? Example in the Kafka connector: [source.Config](https://github.com/ConduitIO/conduit-connector-kafka/blob/7e48b506cce0064301b8a3d9284b8f8e3012a659/source/config.go#L23-L32), [destination.Config](https://github.com/ConduitIO/conduit-connector-kafka/blob/7e48b506cce0064301b8a3d9284b8f8e3012a659/destination/config.go#L26-L45), [common.Config](https://github.com/ConduitIO/conduit-connector-kafka/blob/7e48b506cce0064301b8a3d9284b8f8e3012a659/common/config.go#L31-L46). Notice how `source.Config` and `destination.Config` embed `common.Config`.

Reply:
@anna-cross Did you have time to look at this suggestion?

Reply:
@anna-cross in case you missed it ^

Reply:
Thanks for the bump on this! Adding it!

Reply:
Thanks! Small thing related to this. When changing the configurations, we should also make sure that the parameter stuff is re-generated. To make sure the files are up to date, you can add [this check](https://github.com/ConduitIO/conduit/blob/main/.github/workflows/validate-generated-files.yml) (minus the protobuf stuff). It relies on having a `generate` target in the `Makefile`, like this:
```
generate:
	go generate ./...
```